package.json

```diff
@@ -13,13 +13,12 @@
   "bugs": {
     "url": "https://github.com/mozilla/react-metrics-graphics/issues"
   },
-  "files": ["build"],
-  "dependencies": {
-    "classnames": "^2.1.2",
-    "metrics-graphics": "^2.12.0"
-  },
+  "files": [
+    "build"
+  ],
   "peerDependencies": {
     "metrics-graphics": "^2.11.0",
+    "prop-types": "^15.6.0",
```

Comment:
I've never quite understood `peerDependencies`, so this is probably just be my ignorance asking, but does this need to be a `peerDependency`?

Reply:
Yes, this is a pretty good primer: 

* https://nodejs.org/en/blog/npm/peer-dependencies/
* https://stackoverflow.com/a/34645112

Basically this avoids having prop-types being specified twice in the dependency graph for the project.